desc/compute/_geometry.py

```diff
@@ -182,6 +182,57 @@ def _R0_over_a(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="a_major/a_minor",
+    label="a_{major} / a_{minor}",
+    units="~",
+    units_long="None",
+    description="Elongation",
```

Comment:
Can we specify somewhere that for stellarators this is the maximum elongation across all zeta angles? this is not clear from just looking at the description or the docstrings